From 798b9a8f8075b279c9714beb15351487d46b002b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:08:14 -0400
Subject: [powerpc] add SATA_MV to ppc64_defconfig

Message-id: <1410545655-205645-266-git-send-email-dzickus@redhat.com>
Patchwork-id: 94312
O-Subject: [RHEL7 PATCH 265/626] powerpc: add SATA_MV to ppc64_defconfig
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 5906b0a7018f7debcb41110f64304a6484116dbf
Author: Olof Johansson <olof@lixom.net>
Date:   Fri Jan 3 00:24:19 2014 -0800

    powerpc: add SATA_MV to ppc64_defconfig

    This makes ppc64_defconfig bootable without initrd on pasemi systems,
    most of whom have MV SATA controllers. Some have SIL24, but that driver
    is already enabled.

    Signed-off-by: Olof Johansson <olof@lixom.net>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/configs/ppc64_defconfig b/arch/powerpc/configs/ppc64_defconfig
index 77f681d..ed9bc46 100644
--- a/arch/powerpc/configs/ppc64_defconfig
+++ b/arch/powerpc/configs/ppc64_defconfig
@@ -190,6 +190,7 @@ CONFIG_SCSI_DH_RDAC=m
 CONFIG_SCSI_DH_ALUA=m
 CONFIG_ATA=y
 CONFIG_SATA_SIL24=y
+CONFIG_SATA_MV=y
 CONFIG_SATA_SVW=y
 CONFIG_MD=y
 CONFIG_BLK_DEV_MD=y
-- 
1.7.1